src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.xml

```diff
@@ -0,0 +1,84 @@
+            </summary>
+            <typeparam name="T">The type of value to copy.</typeparam>
+            <param name="destination">The location to copy to.</param>
+            <param name="source">A reference to the value to copy.</param>
+        </member>
+        <member name="M:System.Runtime.CompilerServices.Unsafe.Copy``1(``0@,System.Void*)">
+            <summary>
+            Copies a value of type T to the given location.
+            </summary>
+            <typeparam name="T">The type of value to copy.</typeparam>
+            <param name="destination">The location to copy to.</param>
+            <param name="source">A pointer to the value to copy.</param>
+        </member>
+        <member name="M:System.Runtime.CompilerServices.Unsafe.AsPointer``1(``0@)">
+            <summary>
+            Returns a pointer to the given by-ref parameter.
+            </summary>
+            <typeparam name="T">The type of object.</typeparam>
+            <param name="value">The object whose pointer is obtained.</param>
+            <returns>A pointer to the given value.</returns>
+        </member>
+        <member name="M:System.Runtime.CompilerServices.Unsafe.SizeOf``1">
+            <summary>
+            Returns the size of an object of the given type parameter.
+            </summary>
+            <typeparam name="T">The type of object whose size is retrieved.</typeparam>
+            <returns>The size of an object of type T.</returns>
+        </member>
+        <member name="M:System.Runtime.CompilerServices.Unsafe.As``1(System.Object)">
+            <summary>
+            Casts the given object to the specified type.
+            </summary>
+            <typeparam name="T">The type which the object will be cast to.</typeparam>
+            <param name="o">The object to cast.</param>
+            <returns>The original object, casted to the given type.</returns>
+        </member>
+        <member name="M:System.Runtime.CompilerServices.Unsafe.CopyBlock(System.Void*,System.Void*,System.UInt32)">
+            <summary>
+            Copies bytes from the source address to the destination address.
+            </summary>
+            <param name="destination">The destination address to copy to.</param>
+            <param name="source">The source address to copy from.</param>
+            <param name="numBytesToCopy">The number of bytes to copy.</param>
+        </member>
+        <member name="M:System.Runtime.CompilerServices.Unsafe.InitBlock(System.Void*,System.Byte,System.UInt32)">
+            <summary>
+            Initializes a block of memory at the given location with a given initial value.
+            </summary>
+            <param name="startAddress">The address of the start of the memory block to initialize.</param>
+            <param name="value">The value to initialize the block to.</param>
+            <param name="numBytesToCopy">The number of bytes to initialize.</param>
+        </member>
+    </members>
+</doc>
```

Comment:
It should be possible I suppose, I'm just not sure how to test it. I'll put something that I think is correct for now and we can verify it when there's some way to get Intellisense for C# 7.